scripts/test_translator.py

```diff
@@ -234,8 +234,11 @@ def __init__(self, full_path: str, files: 're.Pattern', keep: List[str], log_lev
         self.target = None
 
         # include the compiler resource directory in compile_commands.json
-        _, stdout, _ = clang["-print-resource-dir"].run(retcode=None)
-        self.clang_resource_dir = " \"-I{}/include\",".format(stdout.strip())
+        if sys.platform == "darwin":
```

Comment:
So we skip setting `clang_resource_dir` on linux, but it only seems to be an issue on Ubuntu 22/clang 15, not later versions.  Should that be checked here?  Although, without this on Ubuntu 24 seems to work as well.

Reply:
Okay, that makes sense. Could you put that in a comment?